<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmember</class>
 <widget class="QDialog" name="addmember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>219</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>434</width>
    <height>219</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>434</width>
    <height>219</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="customerCreationLabel">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Customer Creation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>Address:</string>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="addressLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Interest Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="veryInterestedRadioButton">
         <property name="text">
          <string>Very Interested</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">interestLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="somewhatInterestedRadioButton">
         <property name="text">
          <string>Somewhat Interested</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">interestLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="notInterestedRadioButton">
         <property name="text">
          <string>Not Interested </string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">interestLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Importance Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="keyToHaveRadioButton">
         <property name="text">
          <string>Key</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">importanceLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="niceToHaveRadioButton">
         <property name="text">
          <string>Nice to Have</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">importanceLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="accept">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="interestLevelButtonGroup"/>
  <buttongroup name="importanceLevelButtonGroup"/>
 </buttongroups>
</ui>
